### DISCUSSION TITLE
Rotate Facing of Block

### DISCUSSION THREAD

**Discussion by DasLixou** [Minestom/Minestom#1096] [Help] [answered]
Hello! i just wanna ask how i can rotate the facing of e.g. stairs. so when they are south, they are west, etc.. i couldn't find something like Facing enum..

**Comment by TheMode** [comment, marked as answer]
You have to change the block `facing` property, example:
```java
var block = stairs.withProperty("facing", "east");
```

All properties come from the game, you can press F3 and look at a block to see all its properties. (`facing`, `half`, `shape` and `waterlogged` for a stair)
